### ISSUE TITLE
Nimbus test test_jwt_auth.nim fails on latest nim-stew revision

### ISSUE THREAD

**Issue by kdeme** [status-im/nimbus-eth1#1617]
Bumping nim-stew to the latest version makes `make test` fail:

```
[Suite] EngineAuth: Http/rpc authentication mechanics
    /home/deme/repos/nimbus-eth1/tests/test_jwt_auth.nim(301, 15): Check failed: res.isOk
  [FAILED] JSW/HS256 authentication using shared secret file jwtsecret
    /home/deme/repos/nimbus-eth1/tests/test_jwt_auth.nim(327, 15): Check failed: res.isOk
  [FAILED] JSW/HS256, ditto with protected header variant
subtest no: 6 failed: testjwtauth
```

Printing the specific res.error gives: `Could not connect to remote host`

Bisecting the nim-stew commits, the failure starts occuring from commit https://github.com/status-im/nim-stew/commit/500a61431007a659f6d65b579b8876d0b69f3291

Not sure how/if those are directly related.

**Comment by jangko** [comment]
https://github.com/status-im/nimbus-eth1/blob/0589d49dbc71492d20365fabc4247beed92adb8b/tests/test_jwt_auth.nim#L257

An emergency fix if want to use latest nim-stew is to change `const` to `let`, but I prefer we fix nim-stew.

see status-im/nim-stew#202

**Comment by jangko** [comment]
fixed by #1632, closing.
